client/blocks/eligibility-warnings/index.jsx

```diff
@@ -2,139 +2,71 @@
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NOT_USING_CUSTOM_DOMAIN: {
-			title: translate( 'No custom domain' ),
-			description: translate( 'Your site must use a custom domain to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		NOT_DOMAIN_OWNER: {
-			title: translate( 'Not a custom domain owner' ),
-			description: translate( 'You must be the owner of the primary domain subscription to use this feature.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		SITE_GRAYLISTED: {
-			title: translate( 'Flagged site' ),
-			description: translate( 'This feature is not supported on sites that are not in good standing.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		},
-		SITE_PRIVATE: {
-			title: translate( 'Private site' ),
-			description: translate( 'This feature is not supported on private sites.' ),
-			supportUrl: 'https://support.wordpress.com/'
-		}
-	};
-}
-
-const EligibilityWarnings = props => {
-	const {
-		translate,
-		backUrl,
-		onProceed,
-		siteId,
-		isEligible,
-		eligibilityData,
-		isPlaceholder,
-	} = props;
-
-	const holdMessages = getHoldMessages( translate );
+export const EligibilityWarnings = ( {
+	backUrl,
+	eligibilityData,
+	hasBusinessPlan,
+	isEligible,
+	isJetpack,
+	isPlaceholder,
+	onProceed,
+	siteId,
+	translate,
+} ) => {
+	const context = -1 !== backUrl.indexOf( 'plugins' ) ? 'plugins' : 'themes';
 	const holds = get( eligibilityData, 'eligibilityHolds', [ 'PLACEHOLDER', 'PLACEHOLDER' ] );
 	const warnings = get( eligibilityData, 'eligibilityWarnings', [] );
 
-	const classes = classNames( {
-		'eligibility-warnings__message': true,
-		'eligibility-warnings__placeholder': isPlaceholder,
-	} );
+	const classes = classNames(
+		'eligibility-warnings',
+		{ 'eligibility-warnings__placeholder': isPlaceholder }
+	);
 
 	return (
-		<div className="eligibility-warnings">
+		<div className={ classes }>
 			<QueryEligibility siteId={ siteId } />
-			<SectionHeader label={ translate( 'Conflicts' ) } />
 
-			{ map( holds, ( error, index ) =>
-				<Card key={ index } className={ classes }>
-					<Gridicon icon="notice" className="eligibility-warnings__error-icon" />
-					<div className="eligibility-warnings__message-content">
-						<div className="eligibility-warnings__message-title">
-							{ translate( 'Error: %(title)s', { args: { title: holdMessages[ error ].title } } ) }
-						</div>
-						<div className="eligibility-warnings__message-description">
-							{ holdMessages[ error ].description }
-						</div>
-					</div>
-					<div className="eligibility-warnings__message-action">
-						<Button href={ holdMessages[ error ].supportUrl } target="_blank" rel="noopener noreferrer">
-							{ translate( 'Resolve' ) }
-						</Button>
-					</div>
-				</Card>
-			) }
+			{ 'plugins' === context && ! hasBusinessPlan && ! isJetpack &&
+				<Banner
+					description={ translate( 'Please upgrade to install this plugin.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Upgrade to the Business plan to install plugins.' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).